devtools/conda-envs/test_env.yaml

```diff
@@ -18,15 +18,14 @@ dependencies:
   - python
   - python
   - pyyaml
-  - scipy
+  - scipy <1.14
```

Comment:
Non-blocking comment. Why is it that we are not supporting scipy 1.14+? I wonder if we then have to restrict the version on the conda-forge package and also patch/change all the currently released packages so that older versions don't get installed when environment with scipy 1.14 are resolved. I hope that makes sense.

Reply:
I will relax the pin and see if we have any issues, I am pretty sure it was actually something to do with nose testing or a numba thing

Reply:
Yes, that should be a quick test, great!

Reply:
Oh, I think it was probably related to this, which should've been already fixed on the openmm side. But if we are supporting older OpenMM releases then we might still need the pin. https://github.com/choderalab/openmmtools/issues/743

Reply:
we will need the pin until this gets merged in https://github.com/conda-forge/openmm-feedstock/pull/139 + we patch the older releases